snuba/api/query.py

```diff
@@ -53,15 +54,14 @@ def __init__(
 def raw_query(
     request: Request,
     query: DictClickhouseQuery,
-    client: ClickhousePool,
+    reader: Reader[ClickhouseQuery],
     timer: Timer,
-    stats: MutableMapping[str, Any] = None,
+    stats: Optional[MutableMapping[str, Any]] = None,
```

Comment:
This is unrelated, but noticed it when I was checking the mypy output before and after and realized it'd be an easy fix.